vignettes/including-adam-data-in-teal.Rmd

```diff
@@ -11,60 +11,57 @@ vignette: >
 
 ## Introduction
 
-To include `ADaM` data in a teal app, use the [teal.data::cdisc_data()] function.
+To include `ADaM` data in a teal app, use the `cdisc_data` function.
 
-The [teal.data::cdisc_data()] function allows `teal` applications to include multiple datasets, identifying merge keys
+The `cdisc_data` function allows `teal` applications to include multiple datasets, identifying merge keys
 and providing information to produce R code for reproducibility.
 
 There is an advantage to passing `CDISC` datasets that adhere to `ADaM` standards to these functions in that the code is
 minimized. However, the dataset-related functions also include the flexibility to work with non-standard datasets
 provided that merge keys and the relationship between the datasets are specified.
 
-The examples below illustrate the usage of these different dataset functions for example [teal.data::cdisc_dataset()] and [teal.data::dataset()]. For more information,
+The examples below illustrate the usage of these different dataset functions for example `cdisc_dataset` and `dataset`. For more information,
 see documentation in `teal.data`.
 
-## Join Keys
-
-If you want to learn more about basics of join keys in a non-`ADaM` context, read `vignette("join-keys")`.
+## Keys
 
 Primary keys serve as unique row identifiers in individual datasets and thus need to be specified for each dataset
-and dataset connector. These can be specified on the most general dataset constructor [teal.data::dataset()] as
+and dataset connector. These can be specified on the most general dataset constructor `dataset` as
 shown below.
 
-```{r}
+```{r, message=FALSE}
 library(teal)
 library(scda)
 
-# specify keys x and y on a general dataset
-dataset("x", data.frame(x = c(1, 1:9), y = 2:11, z = 1), keys = c("x", "y"))
-
-# or for ADSL
+# specify keys for ADSL using dataset
 adsl <- synthetic_cdisc_data("latest")$adsl
-dataset("ADSL", adsl, keys = c("STUDYID", "USUBJID"))
+dataset_adsl <- dataset("ADSL", adsl, keys = c("STUDYID", "USUBJID"))
+class(dataset_adsl)
```

Comment:
Do we really want users to care about this? Surely we want them to use cdisc_dataset (so they get a parent) - and they can specify a keys argument there?